loader-tests/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "paths": {
+      "@slowreader/core": ["node_modules/@slowreader/core/index"]
```

Comment:
Should we add it to the global `tsconfig`?

Reply:
Not sure. It's a project specific dep and not all projects use it :man_shrugging: 

If we put it in root `tsconfig` we also have to add it to root `package.json` deps so tsx can resolve it.
And if it's in root then `core` can also reference it? :dizzy:   

I'm leaning towards project-specific but it's your call. You want it in the root `tsconfig`?

Reply:
OK, let's keep it here. I maybe will do a research later